test/images/agnhost/netexec/netexec.go

```diff
@@ -162,7 +167,14 @@ func main(cmd *cobra.Command, args []string) {
 		addRoutes(http.DefaultServeMux, exitCh)
 	}
 
-	go startUDPServer(udpPort)
+	udpBindTo, err := parseAddresses(udpListenAddresses)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, address := range udpBindTo {
+		go startUDPServer(address, udpPort)
+	}
 	if sctpPort != -1 {
 		go startSCTPServer(sctpPort)
```

Comment:
I think that SCTP is stateless and will have the same issue, no?

at least here is failing because only the pod from the other node replies, but not in the same node
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-dualstack-azure-e2e-master-iptables/1362333391973781504